packages/identity-service/src/routes/authentication.js

```diff
@@ -162,6 +162,7 @@ module.exports = function (app) {
       const hasWalletAddressAssociated = existingUser.walletAddress != null
 
       if (!otp) {
+<<<<<<< HEAD
```

Comment:
Artifact from quick merge, will be removed when I revisit the branching logic here